docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,155 @@
+---
+description: Configure the default checkouts process and build custom ones.
+edition: commerce
+---
+
+# Configure checkout
+
+When you work with your Commerce implementation, you can review and modify 
+the checkout configuration.
+
+!!! note "Permissions" 
+
+    When you modufy the workflow configuration, make sure you properly set user [permissions](../permission_use_cases.md#commerce) to the checkout component.
+
+## Checkout workflow
+
+Checkout workflow is configured like other [workflows](../../content_management/workflow/workflow.md) where each 
+transition represents a separate checkout step. 
+
+By default, the checkout process is configured to render each stage based on a separate 
+set of libraries and templates.
+Each checkout step is handled by a controller that you configure in workflow metadata.
+
+Custom workflow implementations are defined under the `framework.workflows` key, 
+and they must support the `Ibexa\Contracts\Checkout\Value\CheckoutInterface`. 
+The default fallback workflow is `ibexa_checkout`, which is prepended at bundle level.
+The `checkout.workflow` parameter is repository-aware.
+
+To customize your configuration, place it in a YAML file, under the `framework.workflows.<your_workflow_name>` key, and reference it with `ibexa.repositories.<your_repository>.checkout.workflow: your_workflow_name`. 
+The system can then identify which of your configured workflows handles the checkout process.
+
+!!! note 
+
+    When you modify or create a controller, to ensure that no user-data is lost, extend the `Ibexa\Bundle\Checkout\Controller\AbstractStep` controller and call the `advance()` method.
+
+Each step configuration includes the following settings:
+
+- `controller` - A mandatory setting pointing to a library that governs the bahavior of the process. It contains all the required business logic and submits the whole step, so that a transition can happen.
```

Comment:
```suggestion
- `controller` - A mandatory setting pointing to a library that governs the behavior of the process. It contains all the required business logic and submits the whole step, so that a transition can happen.
```